src/main/java/plugily/projects/villagedefense/arena/Arena.java

```diff
@@ -207,20 +207,21 @@ public void run() {
   }
 
   public void spawnVillagers() {
-    if(getVillagers().size() > 10) {
-      return;
-    }
     List<Location> villagerSpawns = getVillagerSpawns();
     if(villagerSpawns.isEmpty()) {
       Debugger.debug(Level.WARNING, "No villager spawns set for {0} game won't start", id);
       return;
     }
-    villagerSpawns.forEach(this::spawnVillager);
+
+    int amount = plugin.getConfig().getInt("Villager-Amount", 10);
```

Comment:
You need to add this new config property to `LanguageMigrator` class into the `configUpdate` method.

Reply:
@montlikadani which `case` should I add?